tyk-oss/templates/tests/oss-test.yaml

```diff
@@ -1,17 +1,24 @@
+{{- if .Values.tests.enabled -}}
 apiVersion: v1
 kind: Pod
 metadata:
   name: "{{ include "tyk-oss.fullname" . }}-test-tyk-oss"
   labels:
     chart: {{ include "tyk-oss.chart" . }}
+{{- if .Values.tests.podLabels }}
+{{- toYaml .Values.tests.podLabels | nindent 4 }}
+{{- end }}
   annotations:
     "helm.sh/hook": test
     "helm.sh/hook-delete-policy": hook-succeeded
+{{- if .Values.tests.podAnnotations }}
+{{- toYaml .Values.tests.podAnnotations | nindent 4 }}
+{{- end }}
 spec:
   restartPolicy: Never
   containers:
     - name: health-check-container
-      image: alpine:3.17
+      image: zalbiraw/alpine-curl-jq
```

Comment:
let's use official images.

Reply:
there is no official curl and jq image that I could find. Thats why i did this :( 

we can maintain it as I mentioned above?
https://github.com/TykTechnologies/tyk-charts/pull/357#issuecomment-2480674629

Reply:
Oh, I see. Just in case, can we use `alpine` in the pipeline to avoid potential maintenance work on custom images? Since both your image and Alpine ultimately include similar binaries, this might simplify things on our end.

Also, just so you know, I knew this image: https://gitlab.com/gitlab-ci-utils/curl-jq that is built specifically for API interaction but haven't used it before. it might be useful for your deployments